lib/mix/tasks/ruby.ex

```diff
@@ -0,0 +1,108 @@
+defmodule Mix.Tasks.Thrifter.Ruby do
+  use Mix.Task
+
+  @gen_path "gen/ruby-client"
+  @client_name "test_client"
+
+  @lib_path "#{@gen_path}/lib"
+  @bin_path "#{@gen_path}/bin"
+  @templates_path "templates/ruby"
+
+  @version "0.1.0"
```

Comment:
`mix thrifter.ruby` will be invoked from some server repository (repo in which thrift file lives).
`version` and `client_name` should be taken from that (server) repo `mix.file`:
` version = Mix.Project.config[:version]`